src/components/NavBar/DropDownMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import { faCaretDown, faTools } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import NavBarTools from './NavBarTools';
+import NavBarStyle from './NavBar.module.css';
+import DropDownStyle from '../BasicComponents/DropDownMenu.module.css';
+
+interface MenuProps {
+  platform: {
+    id: string;
+    header: string;
+    style: string;
+  };
+}
+
+const DropDownMenu = ({ platform }: MenuProps) => (
+  <nav>
+    <div className={DropDownStyle.dropdownMenu}>
+      <div className={DropDownStyle.dropdownMenuItem}>
```

Comment:
if there will be no more dropdownMenuItem here, maybe combine it with the drowpdownMenu


if there will be more , arrange that mapping them will be convenient